server/queue/queue-activemq/src/test/java/org/apache/james/queue/activemq/ActiveMQMailQueueBlobTest.java

```diff
@@ -22,48 +22,149 @@
+    static final boolean USE_BLOB = true;
+
+    ActiveMQMailQueue mailQueue;
+    BrokerService broker;
+    MyFileSystem fileSystem;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        fileSystem = new MyFileSystem();
+        broker = createBroker();
+        broker.start();
+        ConnectionFactory connectionFactory = createConnectionFactory();
+        RawMailQueueItemDecoratorFactory mailQueueItemDecoratorFactory = new RawMailQueueItemDecoratorFactory();
+        NoopMetricFactory metricFactory = new NoopMetricFactory();
+        mailQueue = new ActiveMQMailQueue(connectionFactory, mailQueueItemDecoratorFactory, QUEUE_NAME, USE_BLOB, metricFactory);
 
-public class ActiveMQMailQueueBlobTest extends ActiveMQMailQueueTest {
+    }
+
+    protected static BrokerService createBroker() throws Exception {
+        BrokerService broker = new BrokerService();
+        broker.setPersistent(false);
+        broker.setUseJmx(false);
+        broker.addConnector("tcp://127.0.0.1:61616");
+
+        // Enable priority support
+        PolicyMap pMap = new PolicyMap();
+        PolicyEntry entry = new PolicyEntry();
+        entry.setPrioritizedMessages(true);
+        entry.setQueue(QUEUE_NAME);
+        pMap.setPolicyEntries(ImmutableList.of(entry));
+        broker.setDestinationPolicy(pMap);
+        // Enable statistics
+        broker.setPlugins(new BrokerPlugin[]{new StatisticsBrokerPlugin()});
+        broker.setEnableStatistics(true);
+
+        return broker;
+    }
 
-    public static final String BASE_DIR = "file://target/james-test";
-    private MyFileSystem fs;
+    @AfterEach
+    public void tearDown() throws Exception {
+        fileSystem.destroy();
+        broker.stop();
+    }
 
     @Override
-    protected ActiveMQConnectionFactory createConnectionFactory() {
-        ActiveMQConnectionFactory factory = super.createConnectionFactory();
+    public MailQueue getMailQueue() {
+        return mailQueue;
+    }
 
-        FileSystemBlobTransferPolicy policy = new FileSystemBlobTransferPolicy();
-        policy.setFileSystem(fs);
-        policy.setDefaultUploadUrl(BASE_DIR);
-        factory.setBlobTransferPolicy(policy);
+    @Override
+    public ManageableMailQueue getManageableMailQueue() {
+        return mailQueue;
+    }
+
+    @Test
+    @Override
+    @Disabled("JAMES-2295 Disabled as test was dead-locking")
+    public void dequeueCanBeChainedBeforeAck() {
 
-        return factory;
     }
 
+    @Test
     @Override
-    public void setUp() throws Exception {
-        fs = new MyFileSystem();
-        super.setUp();
+    @Disabled("JAMES-2295 Disabled as test was dead-locking")
+    public void dequeueCouldBeInterleavingWithOutOfOrderAck() {
+
     }
 
-    public void tearDown() throws Exception {
-        if (fs != null) {
-            fs.destroy();
-        }
+    @Test
+    @Override
+    @Disabled("JAMES-2301 Per recipients headers are not attached to the message.")
```

Comment:
still not?

Reply:
Still not. Maybe in another PR, if you think it do is important.